java-frontend/src/test/java/org/sonar/java/se/JavaCheckVerifier.java

```diff
@@ -93,6 +95,7 @@
   private static final String DEFAULT_TEST_JARS_DIRECTORY = "target/test-jars";
   private final String testJarsDirectory;
   private final Expectations expectations;
+  private SoftAssertions flowAssertions;
```

Comment:
While this is a nice to have, I don't think it is related to the resolution of the problem raise by this ticket.